modules/dnn/src/layers/elementwise_layers.cpp

```diff
@@ -1129,17 +1187,47 @@ struct MishFunctor : public BaseDefaultFunctor<MishFunctor>
                backendId == DNN_BACKEND_CANN;
     }
 
+    // It is needed by tryQuantize. We can remove this in 5.x.
     inline float calculate(float x) const
     {
-        // Use fast approximation introduced in https://github.com/opencv/opencv/pull/17200
-        if (x >= 8.f)
-        {
-            return x;
-        }
+        x *= (x > -36.73f ? 1.f : 0.f);
```

Comment:
Please add comment and/or reference, where you got -36.73f

Reply:
Added source.